server/container/util/src/test/java/org/apache/james/util/MimeMessageUtil.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations      *
  * under the License.                                           *
  ****************************************************************/
-package org.apache.mailet.base.test;
+package org.apache.james.util;
```

Comment:
server/container is not a good fit, that would basically means mailets depend on a server runtime.
MimeMessage being a javax.mail thing, there's no project to put it in for a good reason : it's purely a library extension.
What about a javax-mail-extension project ?